content/departments/engineering/design/research/user-research-instance.md

```diff
@@ -0,0 +1,34 @@
+# Sourcegraph Cloud instance for user research
+
+One of our tools for carrying out usability testing and getting feedback is our dedicated Sourcegraph Cloud instance for user research.
+
+## Why do we have a dedicated instance for user research?
+
+The product and design team uses a mix of [moderated and unmoderated user research(../index.md) to validate and improve on our work. For some early-stage design research, interactive Figma prototypes do the job. But for many things we want to test, we need to use the real Sourcegraph application, not an interactive prototype.
+
+Sourcegraph.com is not an effective alternative, as the experience on .com doesn’t reflect the actual experience on a customer’s own instance. This in turn influences the research itself so that we may not actually get relevant or valid research insights. Similarly, S2 is internal-only to Sourcegraph, and can’t be accessed outside of the Sourcegraph team.
+
+With our dedicated instance for user research, we can much more accurately explore and validate the customer experience in a like environment.
+
+## Overview of the instance
+
+- The instance is available at https://research.sourcegraph.com.
+- The instance is kept up to date with `main` on an hourly schedule. This means that the research instance can be used while an effort is underway, before each monthly version release.
+- The instance has been populated with a set of open-source repositories “owned” by a fictional organization: Fable Labs. There’s nothing particularly special about these repositories other than representing a variety of languages and complexity, to make them useful for a range of code-related activities.
+- The instance should **not** be considered either stable or permanently stateful. For a variety of reasons, we may choose to either clear all data on the instance or discontinue the instance.
+- Quinn Keast (@quinnkeast) is currently responsible for administering the instance.
+
```

Comment:
We should add some guideline about the content we should create there. So for instance, we should not be creating "Test Batch Change 123" or "Code Insight Example" etc, so that the instance stays clean and as close to reality as possible

Reply:
Also, that if a user creates content as part of a flow, and if it conflicts with other tests, the designer responsible for running those tests should revert the instante to a "clean state" by removing that content.